tox.ini

```diff
@@ -17,3 +17,27 @@ setenv = MONGODB_HOST = localhost
          MONGODB_REPLICASET = 
 commands =
     pytest -s --log-format="%(levelname)%: %(message)s" {posargs}
+
+[testenv:black]
+deps =
+    black
+commands =
+    black --check --diff src/eiffel_graphql_api
```

Comment:
Any reason we're passing just that directory? Doing so leaves out the tests.

Reply:
No reason at all. I missed that one completely.

Reply:
#44